server/aws/nodes.yaml

```diff
@@ -282,6 +266,10 @@ Resources:
           roleName: !Ref 'S3ReadOnlyAccessInstanceProfile'
           buckets:
             - !FindInMap [DeploymentMap, !Ref 'DeploymentType', S3Bucket]
+            - !Join [., [
+                !Ref 'AWS::Region',
+                !FindInMap [DeploymentMap, !Ref 'DeploymentType', S3Bucket]
+              ]]
```

Comment:
The bucket containing the LMDB files needs to be local to the region that the node is deployed in, because otherwise we would be copying a 14GB file cross-region!